components/tidb_query_datatype/src/codec/mysql/set.rs

```diff
@@ -0,0 +1,90 @@
+use crate::codec::data_type::BitVec;
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct Set {
+    data: Vec<u8>,
+    offset: Vec<usize>,
+    value: BitVec,
```

Comment:
https://github.com/pingcap/tidb/blob/master/types/set.go The TiDB implementation just uses `usize` for representing Set, thus it won't have more than 64 bits. So I suggest we just use `usize` in this case. We may modify our RFC later.

Reply:
OK